client/service/src/chain_ops.rs

```diff
@@ -175,6 +181,7 @@ impl<
 					"Stopping after #{} blocks because of an error",
 					link.imported_blocks,
 				);
+				rejected_blocks_number.fetch_add(1, Ordering::Relaxed);
```

Comment:
I think you've confused the location here. This is the import-cli command. In this context we a) don't have metrics activated, nor are we sending them anywhere and b) fail immediately on the error, thus the number would always be `1` and never actually increase.